Tests/Runner/RunnerTest.php

```diff
@@ -2364,6 +2392,76 @@ public function testExecutorInvalidInputMapping(): void
         );
     }
 
+    public function testExecutorFillsInputFilesStateAndInputTableResultOnUserError(): void
```

Comment:
No když jsi to přidal do testExecutorApplicationError, tak proč jsi to nepřidal i do testExecutorUserError a děláš kvůli tomu novej test?